[33mWARNING [0m urllib3.connectionpool:connectionpool.py:810 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002AFADF3DCF0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/03e3be2f8e90960e072b9ad09ed0c5cb/element
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:810 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002AFADF3DE70>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/03e3be2f8e90960e072b9ad09ed0c5cb/element
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:810 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002AFADF3E020>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/03e3be2f8e90960e072b9ad09ed0c5cb/element